.azure-pipelines/scripts/validate_samples.R

```diff
@@ -24,11 +24,21 @@ getPathLeaves <- function(path){
   return(unlist(ret))
 }
 
-validate_samples <- function(directory) {
-
+validate_samples <- function(args) {
+  directory = args[1]
   sample_dirs = getPathLeaves(directory)
 
+  skip_tests = c()
+
+  if (length(args) > 1) {
+    skip_tests <- args[2:length(args)]
```

Comment:
in case of multiple tests, i'd separate by semicolon.

Reply:
you can pass them as separate argument. it looks cleaner

Reply:
what if we need to introduce another argument for other purpose in the future? that very likely will be the case when add validation for vignettes (i.e. add another exclude list for vignettes).  Combing them into a single argument can at least make sure the existing logic and usage won't be changed when adding a new argument. Just my 2 cents.

Reply:
sounds good, updated the script